internal/exec/util/user_group_lookup.c

```diff
@@ -62,10 +62,15 @@ static int user_lookup_fn(lookup_ctxt_t *ctxt) {
 		goto out_err;
 	}
 
-	if(getpwnam_r(ctxt->name, &p, buf, sizeof(buf), &pptr) != 0 || !pptr) {
+	if(getpwnam_r(ctxt->name, &p, buf, sizeof(buf), &pptr) != 0) {
 		goto out_err;
 	}
 
+	if (!pptr) {
+		// successfully found nothing
```

Comment:
According to the docs, this should be setting `name` to NULL.

Reply:
Not sure if that's 100% necessary since `ctxt->res` is allocated in go and passed in (so it's probably zeroed? would need to look that up). But it makes the code clearer and doesn't hurt, so :+1: